storey/targets.py

```diff
@@ -885,33 +877,17 @@ def __init__(
         self._user = user
         self._password = password
         self._database = database
-        self._timeout = timeout
-
-        self._connection = None
-        self._using_websocket = None
 
     def _init(self):
-        import taosrest
+        import taosws
 
         _Batching._init(self)
         _Writer._init(self)
-
-        parsed_url = urlparse(self._url)
-
-        if parsed_url.scheme == "taosws":
-            import taosws
-
-            self._using_websocket = True
+        if self._url.startswith("taosws://"):
```

Comment:
this will be true whether u/p was provided or not, right? if that's the case need to improve the logic.